phpunit.xml.dist

```diff
@@ -1,14 +1,14 @@
 <?xml version="1.0" encoding="UTF-8"?>
 
-<phpunit colors="true" bootstrap="vendor/autoload.php">
+<phpunit colors="true" bootstrap="Tests/bootstrap.php">
     <testsuites>
         <testsuite name="DoctrineBundle for the Symfony Framework">
             <directory>./Tests</directory>
         </testsuite>
     </testsuites>
 
     <php>
-        <env name="SYMFONY_DEPRECATIONS_HELPER" value="ignoreFile=./Tests/baseline-ignore"/>
+        <env name="SYMFONY_DEPRECATIONS_HELPER" value="max[indirect]=9999&amp;ignoreFile=./Tests/baseline-ignore"/>
```

Comment:
@ostrolucky is this fine? So the main CI build will never fail due to deprecations now

Reply:
Or I can restore the behavior with setting it to `weak`? don't mind

Reply:
I am not sure what's the difference between these, but if you believe there is none then yeah keep it. `weak` is no longer mentioned in SF docs, it's something legacy I guess.